packages/graphql-server/src/plugins/useRedwoodTrustedDocuments.ts

```diff
@@ -44,14 +58,21 @@ export const useRedwoodTrustedDocuments = (
   options: RedwoodTrustedDocumentOptions,
 ): Plugin<RedwoodGraphQLContext> => {
   return usePersistedOperations({
+    ...options,
     customErrors: {
       persistedQueryOnly: 'Use Trusted Only!',
       ...options.customErrors,
     },
     getPersistedOperation(sha256Hash: string) {
-      return options.store[sha256Hash]
+      return options.store ? options.store[sha256Hash] : null
     },
     allowArbitraryOperations: async (request) => {
+      if (typeof options.allowArbitraryOperations === 'function') {
+        const result = await options.allowArbitraryOperations(request)
+        if (result === true) {
+          return true
+        }
+      }
       return allowRedwoodAuthCurrentUserQuery(request)
     },
```

Comment:
Okay my only concern then is that the types would have you believe you can use a simple boolean value rather than a function. Unless I'm reading the types wrong of course.

Reply:
Now checks that allowArbitraryOperations is a boolean and if true, allows all arbitrary operations.

If false, then falls through to allow RW operation.

Of, enforces a function to allow operations.